test/examples/active_message.c

```diff
@@ -185,13 +183,9 @@ int main(int argc, char **argv)
         goto out;
     }
 
-    /* Initialize context */
-    status = ucs_async_context_init(&async, UCS_ASYNC_MODE_THREAD);
-    CHKERR_JUMP(UCS_OK != status, "init async context", out);
-
     /* Create a worker object */
-    status = uct_worker_create(&async, UCS_THREAD_MODE_SINGLE, &if_info.worker);
-    CHKERR_JUMP(UCS_OK != status, "create worker", out_cleanup_async);
+    status = uct_worker_create(NULL, UCS_THREAD_MODE_SINGLE, &if_info.worker);
```

Comment:
I removed `ucs_async_context_t` from the example because it uses internal data type.  I plan to do some refactoring of the code related to `<ucs/async/async.h>` to make the type as opaque. This will change UCP interface function `ucs_async_context_init()` behaviour (it will allocate a memory for `async`).
I change this example in this way to separate this work.